tools/measurement/latency/src/types.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface ProwJob {
+  apiVersion: string;
+  kind: string;
+  metadata: {
+    name: string;
+  };
+  spec: {
+    job: string;
+    refs: {
+      pulls?: Array<{ number: number }>;
+    };
+    type: string;
+  };
+  status: {
+    startTime: string;
+    completionTime: string;
+    state: string;
+  };
+}
+
+export interface ReplaceMeasurement extends Array<string | number> {}
+
+export interface Table {
```

Comment:
Similar to my comments above, I think we could put some of those functions above here to be more object oriented.

Reply:
Done.